components/src/core/empty-state/empty-state.component.html

```diff
@@ -1,9 +1,9 @@
-<div class="container" [style.color]="Colors.gray[500]">
-    <div style="line-height: 1">
+<div class="pxb-empty-state">
+    <div class="pxb-empty-state-empty-icon-wrapper">
```

Comment:
do we want to drop the 'wrapper' part of our class names?

Reply:
I put the 'wrapper' part because it's wrapping a ng-content.  It's not the content itself so I think the wrapper part is appropriate.

For styling ng-content I've been following this is class naming convention: 
`ng-content-selector-name-wrapper`

Reply:
sounds good to me, as long as we're consistent